test/extended/router/scoped.go

```diff
@@ -155,18 +151,22 @@ var _ = g.Describe("[Conformance][Area:Networking][Feature:Router]", func() {
 			}
 
 			g.By("checking that the router reported the correct ingress and override")
-			r, err := oc.RouteClient().Route().Routes(ns).Get("route-1", metav1.GetOptions{})
+			r, err := oc.RouteClient().RouteV1().Routes(ns).Get("route-1", metav1.GetOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred())
 			ingress := ingressForName(r, "test-override")
 			e2e.Logf("Selected: %#v, All: %#v", ingress, r.Status.Ingress)
 			o.Expect(ingress).NotTo(o.BeNil())
 			o.Expect(ingress.Host).To(o.Equal(fmt.Sprintf(pattern, "route-1", ns)))
-			external := routev1.RouteIngress{}
-			err = routev1conversions.Convert_route_RouteIngress_To_v1_RouteIngress(ingress, &external, nil)
-			o.Expect(err).NotTo(o.HaveOccurred())
-			status, condition := routedisplayhelpers.IngressConditionStatus(&external, routev1.RouteAdmitted)
-			o.Expect(status).To(o.Equal(corev1.ConditionTrue))
-			o.Expect(condition.LastTransitionTime).NotTo(o.BeNil())
+
```

Comment:
@squeed @deads2k i'm not sure why is this in extended test or what is the purpose of this test?

Reply:
i guess if we want this we can import the conversions from openshift-apiserver once we do the move

Reply:
> i guess if we want this we can import the conversions from openshift-apiserver once we do the move

Let's call that plan never.